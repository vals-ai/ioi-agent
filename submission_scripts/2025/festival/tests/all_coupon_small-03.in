version https://git-lfs.github.com/spec/v1
oid sha256:5199f0dc2371e97070719557956dd9af4fd7ed3211f39d6355932acdd7aeb2ed
size 370
