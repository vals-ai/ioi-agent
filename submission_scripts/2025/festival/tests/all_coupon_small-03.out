version https://git-lfs.github.com/spec/v1
oid sha256:e90b27c50e214298f6d0f6ea141adacd1a052a839947e12319a0a6cbb0be2de0
size 179
